### PR TITLE
chore: update buy_button_link to Osmosis Pool

### PR DESCRIPTION
**Pull request by dpdanpittman**
## Description

Closes: regen-network/regen-registry#1624

With the launch of NCT on Osmosis we are now ready to update the link on the Buy NCT Button.  This code updates href constant to https://app.osmosis.zone/?from=REGEN&to=NCT

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/components/organisms/BasketDetailsActionsBar/BasketDetailsActionsBar.constants.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 export const BUY_BASKET_TOKENS_TOOLTIP =
   'NCT represents the first IBC-compatible carbon token to the Interchain economy, and launches via NCT: Regen and NCT: Osmo pools on the Osmosis exchange.';
 export const BUY_BASKET_TOKENS = 'buy NCT';
-export const BUY_BUTTON_LINK =
-  'https://guides.regen.network/guides/regen-marketplace/bridging/polygon-to-regen-ledger';
+export const BUY_BUTTON_LINK = 'https://app.osmosis.zone/?from=REGEN&to=NCT';
```
Comment (clevinson) [inline web-registry/src/components/organisms/BasketDetailsActionsBar/BasketDetailsActionsBar.constants.ts:4]:
```suggestion
export const BUY_BUTTON_LINK = 'https://app.osmosis.zone/?from=USDC&to=NCT';
```

Comment (clevinson) [reply web-registry/src/components/organisms/BasketDetailsActionsBar/BasketDetailsActionsBar.constants.ts:4]: We should list USDC as the from price in this

**Comment by clevinson** [comment]
Let's hold off on merging until https://app.osmosis.zone/?from=USDC&to=NCT works as expected (it should show NCT as the purchase currency, and USDC as the sell currency)

**Comment by clevinson** [comment]
@wgwz From a release process, should this be merging into `dev` first and then back ported to `main` ?

**Review by haveanicedavid — APPROVED** [review]
utACK

**Comment by wgwz** [comment]
> From a release process, should this be merging into dev first and then back ported to main ?

@clevinson yup! that's right, i'll update the target branch

**Pull request merged**